docker-compose.yml

```diff
@@ -172,7 +172,10 @@ services:
     command: '"0 0 * * * gosu sentry sentry cleanup --days $SENTRY_EVENT_RETENTION_DAYS"'
   nginx:
     << : *restart_policy
+    #expose:
+    #  - '443'
```

Comment:
This instruction is obsolete so let's just remove it

```suggestion
```

Reply:
it is? opps i will test without and confirm it as working.

Reply:
confirmed as not needed